xkit.css

```diff
@@ -368,6 +368,8 @@
 	-moz-user-select: none;
 	-webkit-user-select: none;
 	user-select: none;
+	max-height: 80vh;
+  overflow-y: scroll;
```

Comment:
nit: indentation style mismatch

Reply:
Oh, weird, I guess eslint isn't set to catch that!